opensearch/variables.tf

```diff
@@ -27,6 +27,8 @@ variable "config" {
     search_slow_log_retention_in_days = optional(number)
     es_app_log_retention_in_days      = optional(number)
     audit_log_retention_in_days       = optional(number)
+    password_special_characters       = optional(string)
```

Comment:
```suggestion
    special_characters_allowed_in_password       = optional(string)
```

Reply:
Thats a bit clunky and sounds like it should be a bool. 

Other options:

special_character_list_for_password
password_special_character_list

Reply:
`password_special_character_list`

Reply:
Actually, leave it as is.